site/src/components/Margins/Margins.tsx

```diff
@@ -26,14 +26,11 @@ export const Margins: FC<MarginsProps> = ({
 	return (
 		<div
 			{...divProps}
-			css={{
-				marginLeft: "auto",
-				marginRight: "auto",
-				maxWidth: maxWidth,
-				paddingLeft: sidePadding,
-				paddingRight: sidePadding,
-				width: "100%",
+			style={{
+				"--max-width": `${maxWidth}px`,
+				"--padding": `${sidePadding}px`,
```

Comment:
this is trying to over optimizing again

this pattern generates more code and is slower, because tailwind needs to inline additional classes and it requires the css engine to do a lookup and cast. it's almost never a good idea.